pages/badges.jsx

```diff
@@ -0,0 +1,192 @@
+var React = require('react'),
+    ImageTag = require('../components/imagetag.jsx'),
+    HeroUnit = require('../components/hero-unit.jsx'),
+    IconLinks = require('../components/icon-links.jsx'),
+    IconLink = require('../components/icon-link.jsx'),
+    BadgeVerticalIcon = require('../components/badge-vertical-icon.jsx'),
+    Link = require('react-router').Link,
+    _ = require('underscore');
+
+var BadgesPage = React.createClass({
+    statics: {
+        pageTitle: 'Badges',
+        pageClassName: 'badges'
+    },
+    getInitialState: function () {
+        var dummyBadges = [
+            {
+                key : 1,
```

Comment:
better not to use plain numbers as keys. Using `title` for the key is a safer approach.